out/blocks/wirecard_credit_card_fields.tpl

```diff
@@ -0,0 +1,38 @@
+[{*
```

Comment:
Could you move the file into views/blocks

Reply:
removed, also the remaining items in `/out/blocks` are now in the `/views` dir